core/controllers/collection_viewer.py

```diff
@@ -46,7 +47,7 @@ def test_can_play(self, collection_id, **kwargs):
 class CollectionPage(base.BaseHandler):
     """Page describing a single collection."""
 
-    @require_collection_playable
+    @action_decorators.play_collection
```

Comment:
I thought the name in terms of actions. Eg: 
```
(decorator for action you want to perform)
(block containing code for performing action)
```

_play_collection_ sounds like action and _can_play_collection_ sound like a permission check. Should I change it to _can_play_collection_ (then I'll change likewise for other decorators as well)?